apps/vr-tests/src/stories/Link.stories.tsx

```diff
@@ -51,4 +51,14 @@ storiesOf('Link', module)
     <Link disabled styles={{ root: { fontSize: '14px' } }}>
       I'm rendered as a button because I have no href and am disabled
     </Link>
+  ))
+  .addStory('Underlined', () => (
+    <Link underline styles={{ root: { fontSize: '14px' } }}>
+      I'm rendered as a button because I have no href
```

Comment:
Does it matter if we have a test where the link rendered as `<a>` or not?

Reply:
Just changed that, thanks for letting me know.